### DISCUSSION TITLE
Number of "issues" is increasing - any mid-term plan ?

### DISCUSSION THREAD

**Discussion by adynis** [Koenkk/zigbee2mqtt#22719] [General]
Just a curiosity ... :) 

I appreciate a lot the entire effort of Z2M development team, I use Z2M in HomeAssistant and it's very helpful and here, on github, you can find help & support, which is great.
I observed: In last ~2...3 months the number of issues in github "issues" section increased from 1.3K to 1.5K. (this includes also new device support or feature requests). 
So. ..  **I was wandering**: Is there any mid-term plan to mitigate this? any solution/proposal/hope/expectation to decrease that number (without affecting Z2M quality :) ) (Is HomeAssistant dev team supporting Z2M ? )

**Comment by Koenkk** [comment]
I think one of the main reasons the amount of open issues is increasing is because I've modified the stale bot time from 30 to 180 days.
